src/core/datastore/datastore.js

```diff
@@ -78,7 +78,7 @@ export class DataStore {
    * @param  {Object} [options={}] Options
    * @return {Promise<Object>} The result of clearing the cache.
    */
-  static clearCache(options = {}) {
+  static clearCache() {
```

Comment:
You should remove the information about the `options` in the documentation for the function.